src/VisualStudio/IntegrationTests/CSharp/CSharpAutomaticBraceCompletion.cs

```diff
@@ -1,208 +1,523 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-using System;
-using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
 using Roslyn.Test.Utilities;
-using Roslyn.VisualStudio.IntegrationTests;
 using Roslyn.VisualStudio.Test.Utilities;
+using Roslyn.VisualStudio.Test.Utilities.Input;
 using Xunit;
 
-namespace Roslyn.VisualStudio.CSharp.IntegrationTests
+namespace Roslyn.VisualStudio.IntegrationTests.CSharp
```

Comment:
Isn't this backwards from our normal naming convention? (Example: https://github.com/dotnet/roslyn/blob/master/src/Workspaces/CSharp/Portable/CaseCorrection/CSharpCaseCorrectionService.cs)